chart/k8gb/values.yaml

```diff
@@ -87,7 +87,7 @@ infoblox:
   # -- use SSL
   sslVerify: true
   # -- Request Timeout in secconds
-  httpRequestTimeout: 20
+  httpRequestTimeout: 60
```

Comment:
@jkremser I don't think we should increase the default value because of some local cases, this value can be overridden in a custom values file.